examples/research_projects/ipu-bert/bert.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2021 Graphcore Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import time
+import wandb
```

Comment:
Can we move the wandb import within the `if config.wandb`? This way wandb is not a hard requirements for people who would like to train without it

Reply:
Sorry, just saw this. I've done it.